frontend/src/app/main/ui/workspace/sidebar/options/rows/stroke_row.cljs

```diff
@@ -140,8 +141,18 @@
     [:div {:class (stl/css-case
                    :stroke-data true
                    :dnd-over-top (= (:over dprops) :top)
-                   :dnd-over-bot (= (:over dprops) :bot))
-           :ref dref}
+                   :dnd-over-bot (= (:over dprops) :bot))}
+
+     (when (some? on-reorder)
+       [:*
+        [:div {:ref dref :class (stl/css :reorder)}
+         [:> icon*
+          {:id ic/reorder
+           :class (stl/css :reorder-icon)
+           :aria-hidden true}]]
+        [:hr {:class (stl/css :reorder-separator-top)}]
+        [:hr {:class (stl/css :reorder-separator-bottom)}]])
```

Comment:
I see this being repeated all over, maybe it should be a component itself?

Reply:
I think so. This will also solve the issue with the "refactor" because then the styles will be owned by the component